Makefile.base

```diff
@@ -100,25 +100,25 @@ $(OBJC): $(BINDIR)/$(MODULE)/%.o: %.c $(RIOTBUILD_CONFIG_HEADER_C)
 	$(Q)$(CCACHE) $(CC) \
 		-DRIOT_FILE_RELATIVE=\"$(patsubst $(RIOTBASE)/%,%,$(abspath $<))\" \
 		-DRIOT_FILE_NOPATH=\"$(notdir $<)\" \
-		$(CFLAGS) $(INCLUDES) -MD -MP -c -o $@ $(abspath $<)
+		$(CFLAGS) $(INCLUDES) -MQ $@ -MD -MP -c -o $@ $(abspath $<)
```

Comment:
Not sure if this is necessary, but the docs [1] put apostrophes around the parameter:
```
 -MQ '$(objpfx)foo.o'
```
That might help for cases where `$@` has whitespaces (though currently this can never happen, I suppose).

https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html